emcc.py

```diff
@@ -2345,6 +2348,8 @@ def parse_args(newargs):
   wasm_eh_enabled = False
   skip = False
 
+  compile_only = '-c' in newargs or '-S' in newargs or '-E' in newargs
```

Comment:
this could call a helper function that is shared with the above? then both lines for `compile_only` could be
```python
compile_only = check_compile_only(newargs)
```

(the above also wants the individual values, so this will cause a little more work there, but it seems more maintainable that way to me)